docker-entrypoint.sh

```diff
@@ -20,10 +20,7 @@ fi
 
 if [[ "$APPLY_MIGRATIONS" = "1" ]] && [[ "$SEED_DEVELOPMENT_DATA" = "1" ]]; then
     echo "Seeding initial development data..."
-    ./manage.py seed_data --development
-elif [[ "$APPLY_MIGRATIONS" = "1" ]]; then
```

Comment:
Could some installation somewhere depend on the fact that APPLY_MIGRATIONS also adds the seed data?

Reply:
I guess that's possible. Perhaps this should be marked as a breaking change in the next release?